src/test/data/JsonSerializableAddressBookTest/duplicatePetsAddressBook.json

```diff
@@ -5,12 +5,14 @@
     "ownerName": "Alice Pauline",
     "address": "123, Jurong West Ave 6, #08-111",
     "tagged": [ "friends" ],
-    "diet": "cute oranges"
+    "diet": "cute oranges",
+    "appointment": "Every monday bring to vet at 2pm"
```

Comment:
Hm, the appointment field in this case isn't in the correct form dictated in `VALIDATION_REGEX` of the `Appointment` class. Since the data in this file is meant to test if the app will handle duplicates, not invalid appointment format, maybe it's better to have it in the correct format? Same for some of the other test data as well.